### PR TITLE
[CBRD-24599] Add new csql session command 'SingleLine'

### PR DESCRIPTION
**Pull request by eido5**
http://jira.cubrid.org/browse/CBRD-24599

**Purpose**

- In csql, there are two modes to handle sql statements.

        single line (with semicolon, handling sql each line)
        --no-single-line (handling the sql at once with ;xr or ;r session command)
        

- To csql change mode, we must quit the csql and restart with '--no-single-line' option or not. 
So, we need to switch the both modes while csql is working.
This session command will help user use csql more convenient.

**Implementation**

Add new csql session command.

**Remarks**

N/A

### CODE REVIEW

File: src/executables/csql.c [hunk 1]
```diff
@@ -1439,6 +1439,17 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
...
+
+    case S_CMD_SINGLELINE:
+      if (!strcasecmp (argument, "on"))
+	{
+	  csql_arg->single_line_execution = true;
+	}
+      else if (!strcasecmp (argument, "off"))
+	{
+	  csql_arg->single_line_execution = false;
+	}
```
Comment (kisoo-han) [inline src/executables/csql.c:1451]: When the csql mode is '**_interactive_**', you have to print out current SINGLELINE MODE,
for example
```
csql> ;single
SINGLELINE IS ON
csql> ;single off
SINGLELINE IS OFF
```
Please refer the session command '**AUTOCOMMIT**'.

**Review by kisoo-han — APPROVED** [review]

**Comment by kisoo-han** [comment]
I think its better to include 'SINGLELINE' to the '**csql HELP**', with this PR.

File: src/executables/csql.c (hunk 1 above)
Comment (eido5) [reply src/executables/csql.c:1451]: Thank you. I will ask you review again after I will add the function which is the status of 'SingleLine' mode

File: src/executables/csql.c [hunk 2]
```diff
@@ -1449,6 +1449,9 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
 	{
 	  csql_arg->single_line_execution = false;
 	}
+
+      fprintf (csql_Output_fp, "SINGLELINE IS %s\n", (csql_arg->single_line_execution ? "ON" : "OFF"));
```
Comment (kisoo-han) [inline src/executables/csql.c]: Print only when it is csql interactive (check for the variable **csql_Is_interactive** is **true**).

**Comment by kisoo-han** [comment]
1. update csql help display a line for 'SINGLELINE'.
2. update your repo with base repository to remove circle-ci fails (test_sql, test_sql_ol)

File: src/executables/csql.c (hunk 2 above)
Comment (eido5) [reply src/executables/csql.c]: I understand. I will apply it.

**Review by airnet73 — APPROVED** [review]

**Review by kisoo-han — APPROVED** [review]
Ok, go ahead for other languages.

**Review by Srltas — APPROVED** [review]

**Pull request merged**